src/braket/experimental/autoqasm/types/types.py

```diff
@@ -13,7 +13,9 @@
 
 """AutoQASM types and type utilities."""
 
-from typing import Any, Optional
+from __future__ import annotations
```

Comment:
Confusing, this is not the same annotations as we're mostly talking about here, this is some Python lang thing?